plutus-ledger/src/Ledger/Orphans.hs

```diff
@@ -109,19 +108,12 @@ instance OpenApi.ToSchema JSON.Value where
     declareNamedSchema _ = pure $ OpenApi.NamedSchema (Just "JSON") mempty
 deriving instance OpenApi.ToSchema ann => OpenApi.ToSchema (Kind ann)
 deriving newtype instance OpenApi.ToSchema Ada
-deriving instance OpenApi.ToSchema Tx
-deriving instance OpenApi.ToSchema Certificate
 deriving instance OpenApi.ToSchema DCert
-deriving instance OpenApi.ToSchema TxIn
-deriving instance OpenApi.ToSchema TxInput
-deriving instance OpenApi.ToSchema TxInType
 deriving instance OpenApi.ToSchema ScriptTag
 deriving instance OpenApi.ToSchema RedeemerPtr
 deriving instance OpenApi.ToSchema TxOutRef
 deriving instance OpenApi.ToSchema PV1.TxOut
 deriving instance OpenApi.ToSchema PV2.TxOut
-deriving instance OpenApi.ToSchema TxInputType
-deriving instance OpenApi.ToSchema Withdrawal
```

Comment:
Not sure if it's a good idea to move these instances to definitions back again. If I remember there was an idea to move them to a different package.

Reply:
I didn't check if they were there first. I can move them back with the new instances.